app/models/book.rb

```diff
@@ -1,3 +1,24 @@
 class Book < ApplicationRecord
-  
+
+  has_many :reviews
+
+  def avg_rating
+    reviews.average_rating
```

Comment:
I really like moving this functionality to the Review model. The method is so clearly named that I don't have any confusion about what's happening, even though I don't see the specific details of how it's being calculated.